posts/2020-02-27-Rust-1.41.1.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Announcing Rust 1.41.1"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team has published a new point release of Rust, 1.41.1.
+Rust is a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust 1.41.1 is as easy as:
+
+```console
```

Comment:
This isn't feedback, more of a question. Is `console` now the preferred marker for CLI instructions? Is there any meaningful difference?